roles/openshift_node/tasks/config.yml

```diff
@@ -16,6 +16,16 @@
     sysctl_file: "/etc/sysctl.d/99-openshift.conf"
     reload: yes
 
+# The base OS RHEL with "Minimal" installation option is
+# enabled firewalld serivce by default, it denies unexpected 10250 port.
+# Reference: https://bugzilla.redhat.com/show_bug.cgi?id=1740439
+- name: Disable firewalld service
+  systemd:
+    name: "{{ item }}"
+    enabled: no
+  with_items:
```

Comment:
Please use `loop`.
https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#migrating-from-with-x-to-loop

Reply:
Thank you for your review ! I've replaced `with_items` with `loop` directive.